Acrolinx.Sidebar/AcrolinxSidebar.cs

```diff
@@ -116,34 +124,18 @@ private void SetDefaultClientLocaleIfNotSet()
             }
         }
 
-        private void ShowServerSelectorIfServerAddressNotSet()
-        {
-            if (!string.IsNullOrWhiteSpace(this.ServerAddress))
-            {
-                this.ShowServerSelector = true;
-            }
-        }
-
-        private void HideServerSelectorIfServerAddressParameterSet(string serverAddress)
-        {
-            if (!string.IsNullOrWhiteSpace(serverAddress))
-            {
-                this.ShowServerSelector = false;
-            }
-        }
-
         private void SetDefaultForSidebarLocationAndShowServerSelectorIfLocationNotSet(string serverAddress)
         {
             if (string.IsNullOrWhiteSpace(this.SidebarSourceLocation))
             {
-                if (!string.IsNullOrWhiteSpace(serverAddress))
+                if (!this.ShowServerSelector && (!string.IsNullOrWhiteSpace(serverAddress)))
                 {
                     this.SidebarSourceLocation = getServerAddress(serverAddress) + defaultSidebarServerLocation;
```

Comment:
Then ShowServerSelector is true always. We should remove ShowServerSelector completely.